lib/Dialect/Torch/Transforms/ShapeLibrary.cpp

```diff
@@ -39,11 +39,11 @@ module {
   func @__torch__.torch_mlir.dialects.torch.importer.jit_ir.build_tools.upstream_shape_helpers._copy(%arg0: !torch.list<int>) -> !torch.list<int> {
     %true = torch.constant.bool true
     %0 = torch.prim.ListConstruct  : () -> !torch.list<int>
-    %1 = torch.aten.len.t %arg0 : !torch.list<int> -> !torch.int
+    %1 = torch.aten.len.t %arg0 :  !torch.list<int> ->  !torch.int
```

Comment:
Can you rebase to the latest and rerun `update_shape_lib.sh`. A recent fix from https://github.com/llvm/torch-mlir/pull/687 should avoid adding the extra spaces.